main.py

```diff
@@ -38,17 +38,31 @@
                     print(f"Listening: {artist_name} - {track_name}")
 
             response = requests.get(current_track['item']['album']['images'][0]['url'])
-
-            with open("album.png", "wb") as album:
+            album_blur = finalImage("album.png")
+            with open("album_blur.png", "wb") as album:
```

Comment:
We could get rid of the album_blur if we'll generate the album with the blur already, no?

Reply:
Removed the album_blur part, resolving this thread.